composer.json

```diff
@@ -43,7 +43,7 @@
         "laravel/pint": "^1.0",
         "mockery/mockery": "^1.3.3",
         "orchestra/testbench": "^7.0 || ^8.0",
```

Comment:
```suggestion
        "orchestra/testbench": "^7.0 || ^8.0 || ^9.0",
```

Reply:
Yeah I know, thanks. But it hasn't been tagged yet and I'm still juggling dependencies while things are unreleased. Doing this in the GitHub workflow for now.